Ruddarr/Views/Settings/InstanceView.swift

```diff
@@ -72,6 +78,29 @@ struct InstanceView: View {
         } message: { error in
             Text(error.recoverySuggestionFallback)
         }
+        .alert(
+            isPresented: Binding(
+                get: { self.error != nil },
+                set: { _ in }
+            ),
+            error: error
+        ) { _ in
+            Button("OK") { error = nil }
+        } message: { error in
+            Text(error.recoverySuggestionFallback)
+        }
+    }
```

Comment:
Let's not show an alert when failing, it can just be a message in place of the paths and tapping it will try to load it again.